edx_proctoring/templates/proctored_exam/proctoring_launch_callback.html

```diff
@@ -190,10 +190,7 @@
       var _poll_interval = null;
```

Comment:
I think I'd adjust this code as follows:
1.  Add the following:
```
var _poll_interval_seconds = null;
```
2. Add a `set_poll_interval(new_polling_interval)` function that will set the interval if _poll_interval is null, and clear and set if _poll_interval_seconds is different from new_polling_interval.  We only want to clear for a change.
3. In `ready()`, I'd make an initial call to `set_poll_interval({{default polling interval}})` before calling `poll_exam_status`.
4. Adjust `poll_exam_status` to call `set_poll_interval`.